contrib/sip_proxy/filters/network/source/decoder.cc

```diff
@@ -85,18 +90,17 @@ void Decoder::complete() {
 
 FilterStatus Decoder::onData(Buffer::Instance& data, bool continue_handling) {
   if (continue_handling) {
-    /* means previous handling suspended, continue handling last request,  */
+    /**
+     * means previous handling suspended, continue handling last request
+     */
```

Comment:
Nit: no need for a doc string here and in other places like this.
```suggestion
    // means previous handling suspended, continue handling last request
```

Reply:
done